src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/ExtractCohort.java

```diff
@@ -145,6 +142,12 @@ protected void onStartup() {
             throw new UserException("min-location and max-location should not be used together with intervals (-L).");
         }
 
+        // if there is a avro file, the BQ specific parameters are unnecessary,
+        // but they all are required if there is no avro file
+        if (cohortAvroFileName == null && (projectID == null || cohortTable == null)) {
+            throw new UserException("a project id and cohort table are required if no avro file is provided");
```

Comment:
super small, but for ease of the user, could you provide the argument flags corresponding to project id and cohort table? e.g. `Project id (--project-id) and cohort table (--cohort-extract-table) are required if no avro file is provided.`